Makefile

```diff
@@ -131,7 +133,7 @@ smoke: demo  ## ensure cluster, deploy trust-manager and run smoke tests
 	${BINDIR}/ginkgo -procs 1 test/smoke/ -- --kubeconfig-path ${BINDIR}/kubeconfig.yaml
 
 $(BINDIR)/kubeconfig.yaml: depend ensure-kind _FORCE | $(BINDIR)
-	$(BINDIR)/kind get kubeconfig --name trust > $@
+	$(BINDIR)/kind get kubeconfig --name trust > $@ && chmod 600 $@
```

Comment:
As I understand it, you're spot on for the case when you initially create a cluster for the first time.

With this, I'm trying to help in the case where a `trust` cluster already exists, and the person is coming back to the trust repo after doing some other work (e.g. on cert-manager).

Put another way: with this I'm trying to ensure that if the default kubeconfig was modified _after_ `ensure-kind` was run, we won't rely on the default kubeconfig being unchanging.

We _could_ use `kubectl --context kind-trust` - but we can't control that file and so it could be messed with so it feels a little scarier to rely on it.

Plus having this file lets us use it elsewhere; I used it in the [draft PR](https://github.com/cert-manager/website/pull/1160/files#diff-d81dfaf17a64dfde33b0e443f6b76b70c0c8d6543d9089ea60ffbcb7fa5c8511R171) for the new trust-manager docs, to ensure that users are only talking to the cluster created by the Makefile.

Reply:
That makes sense, thanks for that detailed explanation!